packages/cli/src/consoleParamsReader.ts

```diff
@@ -0,0 +1,121 @@
+ */
+export const parseLogParameters = async (nodeArgV: string[], config: Config): Promise<Either<string, {}>> => {
+    const builder = yargs(nodeArgV);
+    builder.string(NO_WIZARD_LABEL);
+    config.fields.forEach(field => addType(field, builder));
+    addDescription(config.fields, builder);
+    addAlias(config.fields, builder);
+    const argv = builder.argv;
+
+    return await buildLogParameters(config.fields, argv, config.supportedDateFormat);
+};
+
+const buildLogParameters = async (fields: Field[], consoleArguments: {}, supportedDateFormat: string): Promise<Either<string, {}>> => {
+    let returnValues = {};
+    for (let field of fields) {
+        if (consoleArguments[field.name] != null) {
+            if (field.type == "date" && !isValidDate(consoleArguments[field.name], supportedDateFormat)) {
+                return left('Error: Invalid date format. Please enter the date in the following format: ' + supportedDateFormat);
+            }
+            returnValues[field.name] = consoleArguments[field.name];
+        } else {
+            const result = await handleMissingValue(field, consoleArguments, supportedDateFormat, returnValues);
+            if(isLeft(result)){
+                return left(result.left);
+            }
+        }
+    }
+    return right(returnValues);
+};
+
+const handleMissingValue = (async (field: Field, consoleArguments: {}, supportedDateFormat: string, returnValues: {}): Promise<Either<string, true>> => {
+    if (consoleArguments[NO_WIZARD_LABEL] == null) {
+        let fieldValue = await askUserForFieldValue(field, supportedDateFormat);
+        if (fieldValue != null) {
+            returnValues[field.name] = fieldValue;
+        }
+    } else if (consoleArguments[NO_WIZARD_LABEL] == 'to' || (consoleArguments[NO_WIZARD_LABEL] == 't' && !field.optional)) {
+        fillEmptyFieldWithDataType(field, supportedDateFormat, returnValues);
+    } else if (consoleArguments[NO_WIZARD_LABEL] != null && !(consoleArguments[NO_WIZARD_LABEL] == 'to' || consoleArguments[NO_WIZARD_LABEL] == 't')){
+        return left('Invalid usage of --' + NO_WIZARD_LABEL + ": " + consoleArguments[NO_WIZARD_LABEL] + ". Check --help for more info.");
+    }
+    return right(true);
+});
+
+const fillEmptyFieldWithDataType = (field: Field, supportedDateFormat: string, returnValues: {}) => {
+    if(field.enum) {
+        returnValues[field.name] = getEnumString(field);
+    } else {
+        returnValues[field.name] = field.type == "date"? supportedDateFormat : field.type;
+    }
+    if(field.list) {
+        returnValues[field.name] = [returnValues[field.name]];
+    }
+};
+
+const getEnumString = (field: Field): string => {
+    let enumString: string = "";
+    field.enum?.forEach(entry => enumString = enumString + entry + ' | ');
+    return enumString.substr(0, enumString.length -3);
```

Comment:
Die Methode kommt aber glaube ich sowieso weg.
```suggestion
    return field.enum?.join(' | ') ?? '';
```